MAIN/explainer.py

```diff
@@ -0,0 +1,62 @@
+import os
+import asyncio
+from PresentationParser import PresentationParser
+from GPTCommunicator import GPTCommunicator
+from SlideExplanation import SlideExplanation
+from PresentationExporter import PresentationExporter
+
+from dotenv import load_dotenv
+
+# Load environment variables from .env file
+load_dotenv()
+UPLOADS_FOLDER = os.getenv('UPLOADS_FOLDER_PATH')
+
+OUTPUTS_FOLDER = os.getenv('OUTPUTS_FOLDER_PATH')
+
+
+
+async def process_presentation(presentation_path, name_file):
```

Comment:
Write documentation in any of the python's agreed upon formats, an example of one of them
```python
def count_hi_v4(word: str) -> int:
"""
The function counts how many times the string "hi" appears in a given string.
@param word: The given string.
@return: The amount of times the word "hi" appeared.
"""
return word.count("hi")
```